src/Reflection/Adapter/ReflectionType.php

```diff
@@ -7,20 +7,18 @@
 use ReflectionType as CoreReflectionType;
 use Roave\BetterReflection\Reflection\ReflectionIntersectionType as BetterReflectionIntersectionType;
 use Roave\BetterReflection\Reflection\ReflectionNamedType as BetterReflectionNamedType;
+use Roave\BetterReflection\Reflection\ReflectionType as BetterReflectionType;
 use Roave\BetterReflection\Reflection\ReflectionUnionType as BetterReflectionUnionType;
 
 use function array_filter;
 use function array_values;
+use function assert;
 use function count;
 
 abstract class ReflectionType extends CoreReflectionType
 {
-    public static function fromTypeOrNull(BetterReflectionNamedType|BetterReflectionUnionType|BetterReflectionIntersectionType|null $betterReflectionType): ReflectionUnionType|ReflectionNamedType|ReflectionIntersectionType|null
+    public static function fromType(BetterReflectionNamedType|BetterReflectionUnionType|BetterReflectionIntersectionType|null $betterReflectionType): ReflectionUnionType|ReflectionNamedType|ReflectionIntersectionType
```

Comment:
According to infection, this method is not used in a `public` scope? :thinking:

Reply:
It's used in `ReflectionIntersectionType` and `ReflectionUnionType` adapters so marked as `@internal`